core/src/test/scala/pureconfig/PureconfSuite.scala

```diff
@@ -413,6 +413,40 @@ class PureconfSuite extends FlatSpec with Matchers with OptionValues with TryVal
     loadConfig[ConfWithListOfFoo](conf) shouldBe Success(expected)
   }
 
+  it should "be able to load a set of Ints from the system properties" in {
```

Comment:
Thanks for updating all the tests to parse string instead of using system properties!
Nitpick: Please update all of the test descriptions to remove the reference to "system properties".